Internship2022TeamPatri/Internship2022TeamPatri/Base.lproj/LaunchScreen.storyboard

```diff
@@ -11,15 +13,20 @@
             <objects>
                 <viewController id="01J-lp-oVM" sceneMemberID="viewController">
```

Comment:
Please commit only your work, no need to add unnecessary changes to your commits